CHANGELOG.md

```diff
@@ -1,28 +1,44 @@
 # Change Log
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+# <<<<<<< HEAD
+
+## 1.20.1 - 2022-3-25
```

Comment:
```suggestion
## Unreleased
```

We use unreleased for changes that are not released yet, since we don't know what version it will be.

Reply:
Thank you, done. No need to update the changelog at all.